### ISSUE TITLE
Any way to hide the log in the GUI?

### ISSUE THREAD

**Issue by ed789d0** [vishen/go-chromecast#149]
Title.

**Comment by vishen** [comment]
I don't think so unfortunately. It looks like it is set up to always capture the logs: https://github.com/vishen/go-chromecast/blob/master/ui/view_log.go#L11-L32.

It should be possible to pass through a flag to just not execute that function, to disable the logs. But the way it currently is set up it will always show the logs.
